### PR TITLE
Add a simple CI build

### PR DESCRIPTION
**Pull request by ibratoev**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Animal Farm NodeJS CI
```
Comment (veerjd) [inline .github/workflows/ci.yml:1]: Really? haha

Maybe name it something relevant?

**Review by veerjd — CHANGES_REQUESTED** [review]
Change the name

I don't know how to read the rest though, mostly, so otherwise good

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+on:
+  push:
+    branches:
+      - main
```
Comment (veerjd) [inline .github/workflows/ci.yml:5]: branch is master, not main too

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  pull_request:
+    branches:
+      - main
```
Comment (veerjd) [inline .github/workflows/ci.yml:8]: branch is master, not main too

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (veerjd) [inline .github/workflows/ci.yml:11]: Anything above Ubuntu 20.04.5 LTS broke my bot. Is latest above 20.04.5?

**Review by veerjd — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 4 above)
Comment (ibratoev) [reply .github/workflows/ci.yml:11]: That is strange and unexpected...

File: .github/workflows/ci.yml (hunk 3 above)
Comment (ibratoev) [reply .github/workflows/ci.yml]: Good catch - I opened this PR a bit prematurely :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ibratoev) [reply .github/workflows/ci.yml]: Yeah - opened the PR that soon by mistake :)

File: .github/workflows/ci.yml (hunk 4 above)
Comment (veerjd) [reply .github/workflows/ci.yml:11]: Naybe we set a version to not have to worry about future incompatibilities?

**Review by veerjd — APPROVED** [review]

**Pull request merged**